features/support/commit_helpers.rb

```diff
@@ -76,15 +76,22 @@ def committed_files sha
 
 
 # Returns the commits in the currently checked out branch
+#
+# rubocop:disable MethodLength
 def commits_for_branch branch_name
   array_output_of("git log #{branch_name} --oneline").map do |commit|
     sha, message = commit.split(' ', 2)
-
-    unless message == 'Initial commit'
-      { branch: branch_name, message: message, file_name: committed_files(sha) }
-    end
+    next if message == 'Initial commit'
+    filenames = committed_files sha
+    {
+      branch: branch_name,
+      message: message,
+      file_name: filenames,
+      file_content: content_of(file: filenames[0], in_branch: branch_name)
```

Comment:
This should be `content_of(file: filenames[0], for_sha: sha)` as we are getting the file at the commit and not at the tip of the branch. This still works because we never change a file within a branch but logically we could.